examples/EA/SOO/TSP.tpp

```diff
@@ -243,21 +238,21 @@ int main( int argc, char ** argv ) {
 	}
 
 	// Mating selection operator
-	shark::RouletteWheelSelection rws;
+	RouletteWheelSelection rws;
 	// Variation (crossover) operator
-	shark::PartiallyMappedCrossover pmc;    
+	PartiallyMappedCrossover pmc;    

```

Comment:
I would really rename this variable into pmx, since that's the usual abbreviation of partially mapped crossover.